tests/www/test_utils.py

```diff
@@ -148,6 +161,34 @@ def test_open_maybe_zipped_archive(self, mocked_ZipFile, mocked_is_zipfile):
         (args, kwargs) = instance.open.call_args_list[0]
         self.assertEqual('deep/path/to/file.txt', args[0])
 
+    def test_render_example_dagsself(self):
```

Comment:
Really nice. Maybe we should do the same for ALL example dags in a loop ! That would be awesome if we could do that!

Reply:
A good idea. I added this solution ..